mllm/core/aops/GroupedQueryAttentionDecodeOp.hpp

```
@@ -0,0 +1,36 @@
+
+  void load(const ParameterFile::ptr_t& ploader) override;
+
+  void trace(void* trace_context, const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+
+  void forward(const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+
+  void reshape(const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+
+  void setup(const std::vector<Tensor>& inputs, std::vector<Tensor>& outputs) override;
+
+  inline const GroupedQueryAttentionDecodeOpOptions& options() const { return options_; }
```

Clang (14.0.6)

[error] function 'options' should be marked [[nodiscard]]

(modernize-use-nodiscard,-warnings-as-errors)